### ISSUE TITLE
Doesn't work for Windows builds

### ISSUE THREAD

**Issue by TonyHoyleRps** [ammariqais/SkiaForUnity#15]
Android, iOS and the Editor seem to work.  Windows compile bombs out with the following:

```
## Script Compilation Error for: Csc Library/Bee/artifacts/1900b0aP.dag/SkiaSharp.Unity.dll (+2 others)
## CmdLine: "C:\Program Files\Unity\Hub\Editor\2022.3.1f1\Editor\Data\NetCoreRuntime\dotnet.exe" exec "C:/Program Files/Unity/Hub/Editor/2022.3.1f1/Editor/Data/DotNetSdkRoslyn/csc.dll" /nostdlib /noconfig /shared "@Library/Bee/artifacts/1900b0aP.dag/SkiaSharp.Unity.rsp" "@Library/Bee/artifacts/1900b0aP.dag/SkiaSharp.Unity.rsp2"
## Output:
SkiaSharp\Runtime\SkottiePlayer.cs(6,29): error CS0234: The type or namespace name 'Skottie' does not exist in the namespace 'SkiaSharp' (are you missing an assembly reference?)
SkiaSharp\Runtime\SkottiePlayer.cs(26,11): error CS0246: The type or namespace name 'SKCanvas' could not be found (are you missing a using directive or an assembly reference?)
SkiaSharp\Runtime\SkottiePlayer.cs(27,11): error CS0246: The type or namespace name 'SKRect' could not be found (are you missing a using directive or an assembly reference?)
SkiaSharp\Runtime\SkottiePlayer.cs(29,11): error CS0246: The type or namespace name 'SKImageInfo' could not be found (are you missing a using directive or an assembly reference?)
SkiaSharp\Runtime\SkottiePlayer.cs(30,11): error CS0246: The type or namespace name 'SKSurface' could not be found (are you missing a using directive or an assembly reference?)

```
I tried simply removing that file and it caused other errors.. so presumably it's needed.

**Comment by ammariqais** [comment]
@TonyHoyleRps thank you for reporting this bug, should be solved with the latest release

https://github.com/ammariqais/SkiaForUnity/releases/tag/1.0.2

please update SkiaForUnity Package to 1.0.2

**Comment by ammariqais** [comment]
solved on 1.0.2
